.env

```diff
@@ -0,0 +1,5 @@
+﻿process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+NASA_API_KEY=mQsTWmmlkdFaYXnPcOcUVqyMyS2EYkngoxTfm9kA
+NASA_API_URL=https://api.nasa.gov/neo/rest/v1/feed
+PORT=4000
```

Comment:
You can fix this in the next PR
- Don't mess up the env file and add it to gitignore
- Add .env.example with exactly the same keys, but without real data
- Add only the base url for NASA_API_URL=https://api.nasa.gov